charts/microgateway/values.yaml

```diff
@@ -181,60 +181,60 @@ ingress:
     # nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
     # nginx.ingress.kubernetes.io/proxy-ssl-verify: "true"
     # nginx.ingress.kubernetes.io/proxy-ssl-secret: secretName
-  # ingress.labels -- Additional labels to add on the Microgateway ingress
+  # ingress.labels -- Additional labels to add on the Microgateway ingress.
   labels: {}
-  # ingress.hosts -- List of ingress hosts
+  # ingress.hosts -- List of ingress hosts.
   hosts:
     - virtinc.com
-  # ingress.path -- Path for the ingress
+  # ingress.path -- Path for the ingress.
   path: /
-  # ingress.targetPort -- Target port of the service (`http`, `https` or `<number>`)
+  # ingress.targetPort -- Target port of the service (`http`, `https` or `<number>`).
   targetPort: http
 
-  # ingress.tls -- [Ingress TLS](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls) configuration
+  # ingress.tls -- [Ingress TLS](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls) configuration.
   tls: []
   #  - secretName: virtinc-tls
   #    hosts:
   #      - virtinc.com
 
 ## OpenShift route configuration.
 ## ref: https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html
-# route -- [Openshift Route](#openshift-route)
+# route -- [Openshift Route](#openshift-route).
```

Comment:
Remove the dot. 
"Ein- oder Zweiwortbenennungen bekommen keinen Punkt am Ende"

Reply:
Done